cifsadmin/user_admin.c

```diff
@@ -148,12 +148,12 @@ static void __sanity_check(char *pswd_hash, char *pswd_b64)
 
 	if (!pass) {
 		pr_err("Unable to decode NT hash\n");
-		exit(1);
+		exit(EXIT_FAILURE);
```

Comment:
exit(1) and exit(EXIT_FAILURE) are same things, because EXIT_FAILURE == 1.
But OK.

-ss

Reply:
Yeah, just a consistency update.